### ISSUE TITLE
query - but how?

### ISSUE THREAD

**Issue by VinzentE** [APIs-guru/graphql-voyager#25]
Hi there,

I love the tool! Very nice way of visualizing a schema. I am quite new to GraphQL and run into an issue:

Whenever I try to create some sort of **introspection query** and copy it into the "custom schema" field, I am presented with **different errors** like "TypeError: Cannot read property 'map' of undefined", "TypeError: Cannot read property 'types' of undefined" or similar. Even the official examples on http://graphql.org/learn/introspection/ cause errors. 
The only "valid" result I could get working was with the query https://gist.github.com/craigbeck/b90915d49fda19d5b2b17ead14dcd6da on the github API https://developer.github.com/v4/explorer/ 

So in other words:
How does my **introspection query** have to **look like or comply with** to work with graphql-voyager?

Thanks up front!

**Comment by IvanGoncharov** [comment]
Hi @VinzentE,

> How does my introspection query have to look like or comply with to work with graphql-voyager?

You need to clink on `Copy Introspection Query` link and it will be automatically copied into the clipboard. See this picture:
![image](https://user-images.githubusercontent.com/8336157/27255192-f3fb3908-53a1-11e7-91a6-89ff289a8b83.png)
